auto_gptq/modeling/_base.py

```diff
@@ -169,7 +173,8 @@ def __init__(
         is_triton_backend: bool = False,
         injected_fused_attention: bool = False,
         injected_fused_mlp: bool = False,
-        trainable: bool = False
+        trainable: bool = False,
+        is_marlin_format: bool = False,
```

Comment:
It would be cleaner if we can avoid adding `is_marlin_format` here as an arg, it should just be in the quantize_config no?

Reply:
So having the quant_type just be “marlin”?

that would help me on the vLLM side

Reply:
If that’s cool with you, I’ll update @fxmarty

Reply:
If that’s cool with you @fxmarty , I will update

Reply:
Oh I see the question, nvm.

the reason I had this argument is that I needed some way to detect that the model was marlin format during save_pretrained. It makes sense to have that be in the quant config I agree

Reply:
Will make the change

Reply:
Yeah what I meant is that at the point where we call `BaseGPTQForCausalLM.__init__`, I think the `quantize_config` should have a key specifying the marlin format.

Reply:
makes sense, will try to finalize this soon